c/src/lib.rs

```diff
@@ -280,6 +282,26 @@ pub struct FoxgloveServerOptions<'a> {
             responder: *mut FoxgloveFetchAssetResponder,
         ),
     >,
+
+    /// Context provided to the `sink_channel_filter` callback.
+    pub sink_channel_filter_context: *const c_void,
```

Comment:
For my understanding: in the context (heh) of a WebSocket server, when would this be a different context from the one used to construct the server?